src/definitions.yaml

```diff
@@ -312,9 +317,45 @@ paths:
   /blocks/{hash_or_number}/addresses:
     $ref: ./paths/api/blocks/addresses/index.yaml
 
+  # genesis
+
   /genesis:
     $ref: ./paths/api/genesis/index.yaml
 
+  # governance
+
+  /governance/dreps:
+    $ref: ./paths/api/governance/dreps/index.yaml
+
+  /governance/dreps/{drep_id}:
+    $ref: ./paths/api/governance/dreps/{drep_id}/index.yaml
+
+  /governance/dreps/{drep_id}/delegators:
+    $ref: ./paths/api/governance/dreps/{drep_id}/delegators.yaml
+
+  /governance/dreps/{drep_id}/metadata:
+    $ref: ./paths/api/governance/dreps/{drep_id}/metadata.yaml
+
+  /governance/dreps/{drep_id}/updates:
+    $ref: ./paths/api/governance/dreps/{drep_id}/updates.yaml
+
+  /governance/dreps/{drep_id}/votes:
+    $ref: ./paths/api/governance/dreps/{drep_id}/votes.yaml
+
+  /governance/proposals:
+    $ref: ./paths/api/governance/proposals/index.yaml
+
+  /governance/proposals/{tx_hash}/{cert_index}:
+    $ref: ./paths/api/governance/proposals/{tx_hash}/{cert_index}/index.yaml
+
```

Comment:
Missing `/governance/proposals/{tx_hash}/{cert_index}/parameters` 
(`./paths/api/governance/proposals/{tx_hash}/{cert_index}/parameters.yaml`)

Reply:
Yes, that is a proposal that contains new protocol parameters.